Shuttle-back/src/main/java/com/shuttle/driver/DriverDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.shuttle.driver;
+
+import com.shuttle.user.UserDTO;
+
+public class DriverDTO extends UserDTO {
```

Comment:
I think it's better not to do inheritance in DTOs, we'll have redundant code but since it maps to JSON, it makes more sense.